marked-element.html

```diff
@@ -142,9 +142,9 @@
         value: null
       },
       /**
-       * Sanitize the output. Ignore any HTML that has been input.
+       * Do not sanitize the output. Default ignores any HTML that has been input.
```

Comment:
Better phrasing:

Disable sanitization. Default: Ignore any HTML tags contained within the input.